src/literal-escaper/Cargo.toml

```
@@ -0,0 +1,4 @@
+[package]
+name = "literal-escaper"
+version = "0.0.1"
+edition = "2021"
```

```suggestion
edition = "2021"

[dependencies]
std = { version = '1.0.0', optional = true, package = 'rustc-std-workspace-std' }

[features]
rustc-dep-of-std = ["dep:std"]
```

and then proc_macro needs to enable the rustc-dep-of-std feature.